<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PainLocalWindow</class>
 <widget class="QDialog" name="PainLocalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayoutMain" rowstretch="2,1,1,1,1,2,1,1,2,1,1,1" columnstretch="0,1" rowminimumheight="2,1,1,1,1,2,1,1,2,1,1,1" columnminimumwidth="0,1">
     <item row="6" column="0">
      <widget class="QLabel" name="labelIntensity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Интенсивность:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelCharacter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Характер:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSide">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сторона:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="labelLocaliztion">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Локализация:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="labelPartOfTheBodyIrradiation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Часть тела:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxSurface">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="comboBoxPartOfTheBodyIrradiation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Нет</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Спина</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Тело</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Рука</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Нога</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Голова</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelClarification">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Уточнение:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboBoxCharacter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Приступообразная</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Острая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Пароксимальная</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Пульсирующая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Тупая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Колючая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Режущая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Стреляющая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Давящая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сжимающая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Рвущая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Дергающая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сверлящая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Жгучая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Фантомная</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Реактивная</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxClarification">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QComboBox" name="comboBoxLocaliztion">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxPartOfTheBodyLocalization">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Нет</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Спина</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Тело</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Рука</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Нога</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Голова</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPartOfTheBodyLocalization">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Часть тела:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxSide">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Нет</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Левая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Правая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>И левая и правая</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBoxIntensity">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Слабая</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Средняя</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сильная</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Невыносимая</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelSurface">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Поверхность:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="labelIrradiation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Иррадиация:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="labelCharacteristic">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Характеристика:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="labelLocalizationTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Локализация:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QPushButton" name="pushOk">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ОК</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <spacer name="horizontalSpacerBottom">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
